src/components/Nav/Nav.js

```diff
@@ -1,8 +1,41 @@
-import React, { Component } from 'react';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.scss';
 
-export class Nav extends Component {
+class Nav extends React.Component {
   render() {
-    return <div>Nav</div>;
+    return (
+      <nav className="navbar">
+        <div className="navLogo">
+          <Link to="/login">
+            <img
+              alt="logo"
+              className="logoImg"
+              src="http://localhost:3000/images/logo.png"
```

Comment:
```suggestion
              src="/images/logo.png"
```
localhost:3000은 안적어줘도 됩니다! "/"로 시작할 경우 자동으로 현재 서버가 실행중인 url을 인식합니다!

Reply:
수정 완료했습니다!!😎